pkgconfig.mk

```diff
@@ -2,11 +2,11 @@
 # To be used to generate capstone.pc for pkg-config
 
 # version major & minor 
-PKG_MAJOR = 5
+PKG_MAJOR = 4
 PKG_MINOR = 0
 
 # version bugfix level. Example: PKG_EXTRA = 1
-PKG_EXTRA = 0
+PKG_EXTRA = 2
 
 # version tag. Examples: rc1, b2, post1
-PKG_TAG =
+PKG_TAG = 0
```

Comment:
should this be `gt` or something like that?

Reply:
Not all of the packages can use the gt prefix for that component of the version number, so I was just going to have this be the number alone and append it where we are able.

Reply:
ok